index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+    <div class="container">
+        <div class="row">
+            <h3 id="heading">freeCodeCamp Survey Form</h3>
+            <p id="sub-heading">Thank you for taking the time to help us improve the platform</p>
+            <div class="col-md-6">
+                <form>
+                    <div class="mb-3">
+                        <label for="name" class="form-label">Name</label>
+                        <input type="text" class="form-control" id="name" placeholder="Enter your name" required>
+                    </div>
+                    <div class="mb-3">
+                        <label for="name" class="form-label">Email</label>
+                        <input type="email" class="form-control" id="email" placeholder="Enter your Email" required>
+                    </div>
```

Comment:
- [ ] @Duretitibesso3 nice review, the comment is not clear, maybe you wanted to inform the developer to use semantic tags like `<section>` instead of `<div>` tag. Kindly edit it.